Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Person
+    {
+        private string _name;
+        public string GetName()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetName' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5116146)